layouts/partials/footer.html

```diff
@@ -67,12 +67,11 @@
     <script src="{{"js/learn.js" | relURL}}{{ if not .Site.Params.disableAssetsBusting }}?{{ now.Unix }}{{ end }}"></script>
     <script src="{{"js/hugo-learn.js" | relURL}}{{ if not .Site.Params.disableAssetsBusting }}?{{ now.Unix }}{{ end }}"></script>
     {{ if (or (and (ne .Params.disableMermaid nil) (not .Params.disableMermaid)) (not .Site.Params.disableMermaid)) }}
-        {{ if isset .Params "mermaidurl" }}
-            <script src="{{ .Params.mermaidurl }}"></script>
-        {{ else if isset .Site.Params "mermaidurl" }}
-            <script src="{{ .Site.Params.mermaidurl }}"></script>
+        {{ if isset .Params "customMermaidURL" }}
```

Comment:
As mentioned in the [hugo documentation](https://gohugo.io/functions/isset/), this will not work, because the key needs to be lower case!

Reply:
Is this fixed? Just wondering whether I should patch this PR on my site or is it not stable yet?